rellic/AST/IRToASTVisitor.cpp

```diff
@@ -435,6 +448,30 @@ void IRToASTVisitor::VisitFunctionDecl(llvm::Function &func) {
   decl->getAsFunction()->setParams(params);
 }
 
+void IRToASTVisitor::visitIntrinsicInst(llvm::IntrinsicInst &inst) {
+  DLOG(INFO) << "visitIntrinsicInst: " << LLVMThingToString(&inst);
+
+  // NOTE(artem): As of this writing, rellic does not do anything
+  // with debug intrinsics and debug metadata. Processing them to C
+  // is useless (since there is no C equivalent anyway).
+  // All it does it lead to triggering of asserts for things that are
+  // low-priority on the "to fix" list
+
+  if (llvm::isDbgInfoIntrinsic(inst.getIntrinsicID())) {
+    DLOG(INFO) << "Skipping debug data intrinsic";
+    return;
+  } else if (IsAnnotationIntrinsic(inst.getIntrinsicID())) {
```

Comment:
We can skip the `else` here, since the `if` block returns.

Reply:
Fixed